### PR TITLE
Create Executive vote - October 19, 2022.md

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
Adds 2022-10-19 exec copy

### CODE REVIEW

File: governance/votes/Executive vote - October 19, 2022.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+The DC-IAM parameters will be set as follows:
+* [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line): **250 million DAI**.
+* [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): **50 million DAI**.
+* [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): **1 week (604,800 seconds)**.
+
+For more information on the DC-IAM, please see the documentation [here](https://manual.makerdao.com/module-index/module-dciam).
+
+### GUSD PSM Parameter Changes
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmYffkvR#vote-breakdown) the following parameter changes will be made to the GUSD PSM (PSM-GUSD-A) if this executive proposal passes:
```
Comment (blimpa) [inline governance/votes/Executive vote - October 19, 2022.md:47]:
```suggestion
As per this successful governance [poll](https://vote.makerdao.com/polling/QmYffkvR#vote-breakdown), the following parameter changes will be made to the GUSD PSM (PSM-GUSD-A) if this executive proposal passes:
```

File: governance/votes/Executive vote - October 19, 2022.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Parameter changes for the GUSD PSM including raising the Maximum Debt Ceiling, raising the Target Available Debt, and raising the Fee Out, detailed below.
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- A Debt Ceiling Instant Access Module with the parameters listed below will be added to the RWA-007-A Vault.
```
Comment (blimpa) [inline governance/votes/Executive vote - October 19, 2022.md:22]:
```suggestion
- A Debt Ceiling Instant Access Module (DC-IAM) with the parameters listed below will be added to the RWA-007-A Vault.
```

**Review by blimpa — CHANGES_REQUESTED** [review]
LGTM

File: governance/votes/Executive vote - October 19, 2022.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: Template - [Executive Vote] Enable DC-IAM for RWA-007-A (Monetalis Clydesdale), GUSD PSM Parameter Changes - October 13 , 2022
+summary: Enables the Debt Ceiling Instant Access Module for RWA-007-A, parameter changes for the GUSD PSM.
```
Comment (blimpa) [inline governance/votes/Executive vote - October 19, 2022.md:3]:
```suggestion
summary: Enabling the Debt Ceiling Instant Access Module for RWA-007-A, parameter changes for the GUSD PSM.
```

Comment (blimpa) [reply governance/votes/Executive vote - October 19, 2022.md:3]: For consistency ¯\_(ツ)_/¯

Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - October 19, 2022.md]: fixed in 76e6867d23308ab4fc31190b816f5acaa73b60f4

File: governance/votes/Executive vote - October 19, 2022.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Parameter changes for the GUSD PSM including raising the Maximum Debt Ceiling, raising the Target Available Debt, and raising the Fee Out, detailed below.
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- A Debt Ceiling Instant Access Module (DC-IAM) with the parameters listed below will be added to the RWA-007-A Vault.
```
[the remark this answers is not in the document]
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - October 19, 2022.md:22]: We tend to prefer "Human-readable" language for exec copy wherever possible. I can change it to Vault type though which is probably more accurate.

**Pull request merged**